src/crate/theme/vendor/rtd_compat/extension.py

```diff
@@ -1,255 +0,0 @@
-                    subproject["canonical_url"],
-                ),
-            )
-    except Exception:
-        logger.warning(
-            "An error ocurred when hitting API to fetch project/subprojects. Defaulting to an empty list.",
-            exc_info=True,
-        )
-        subprojects = []
-
-    # Add project information to the template context.
-    context = {
-        'html_theme': config.html_theme,
-        'current_version': os.environ.get("READTHEDOCS_VERSION_NAME"),
-        'version_slug': version_slug,
-
-        # NOTE: these are used to dump them in some JS files and to build the URLs in flyout.
-        # However, we are replacing them with the new Addons.
-        # I wouldn't include them in the first version of the extension.
-        # We could hardcode them if we want, tho.
-        #
-        # 'MEDIA_URL': "{{ settings.MEDIA_URL }}",
-        # 'STATIC_URL': "{{ settings.STATIC_URL }}",
-        # 'proxied_static_path': "{{ proxied_static_path }}",
-
-        'PRODUCTION_DOMAIN': production_domain,
-        'versions': versions,
-        "downloads": downloads,
-        "subprojects": subprojects,
-
-        'slug': project_slug,
-        'rtd_language': os.environ.get("READTHEDOCS_LANGUAGE"),
-        'canonical_url': os.environ.get("READTHEDOCS_CANONICAL_URL"),
-
-        # NOTE: these seem to not be used.
-        # 'name': u'{{ project.name }}',
-        # 'analytics_code': '{{ project.analytics_code }}',
-        # 'single_version': {{ project.is_single_version }},
-        # 'programming_language': u'{{ project.programming_language }}',
-
-        'conf_py_path': conf_py_path,
-        # Used only for "readthedocs-sphinx-ext" which we are not installing anymore.
-        # 'api_host': '{{ api_host }}',
-        # 'proxied_api_host': '{{ project.proxied_api_host }}',
-
-        'github_user': github_user,
-        'github_repo': github_repo,
-        'github_version': os.environ.get("READTHEDOCS_GIT_IDENTIFIER"),
-        'display_github': github_user is not None,
-        'bitbucket_user': bitbucket_user,
-        'bitbucket_repo': bitbucket_repo,
-        'bitbucket_version': os.environ.get("READTHEDOCS_GIT_IDENTIFIER"),
-        'display_bitbucket': bitbucket_user is not None,
-        'gitlab_user': gitlab_user,
-        'gitlab_repo': gitlab_repo,
-        'gitlab_version': os.environ.get("READTHEDOCS_GIT_IDENTIFIER"),
-        'display_gitlab': gitlab_user is not None,
-        'READTHEDOCS': True,
-        'using_theme': (config.html_theme == "default"),
-        'new_theme': (config.html_theme == "sphinx_rtd_theme"),
-        'source_suffix': ".rst",
-        'ad_free': False,
-        'docsearch_disabled': False,
-
-        # We don't support Google analytics anymore.
-        # See https://github.com/readthedocs/readthedocs.org/issues/9530
-        'user_analytics_code': "",
-        'global_analytics_code': None,
-
-        'commit': os.environ.get("READTHEDOCS_GIT_COMMIT_HASH")[:8],
-    }
```

Comment:
We'd need to check this whole bunch, if we also set all context variables accordingly?
How would you derive any actions from this? Let's discuss?